### ISSUE TITLE
Highly confusing local install instruction in README

### ISSUE THREAD

**Issue by zach-is-my-name** [ethereum/remix-project#473]
1.   > https://github.com/ethereum/remix-project/releases also gives others ways to use Remix locally. Please check it out.
  - Currently: User is presented with a .zip and the source.  When you download / uncompress the .zip the user is blocked without further instructions 
- Recommended: If you're not issuing binaries, at least provide instructions how to run on the Releases page (if you're directing people to Releases in the first place, as in the above quote, they should be able to complete the flow from there) See how Ganache-cli does it for example https://github.com/trufflesuite/ganache-cli/releases/tag/v6.12.0

2. > Offline Usage The gh-pages branch has always the latest stable build of Remix. It also contains a ZIP file with the entire build. Download it to use offline.
  - Same as above.  And "Download it to use offline" is not a sufficient instruction for *running*

3. > INSTALLATION: ...  
  - More confusion... what is the difference between "Offline Usage" and "Installation" ? 
  - If they are assumed to be the same, the actual instructions are different 

In summary there seem to be 3 separate instructions to do what I want to do - that is, run Ethereum Remix offline, locally - and only the 3rd and final seems sufficient (albeit annoying with the nx dependency)  

You've built a powerful suite of tools, and there is strong case to make the desktop version a first class citizen (starting with enabling smooth import of local files to the Remix suite rather than this multistep remixd business which has been labeled alpha forever, and continuing with privacy concerns about pushing proprietary contracts to a relatively unknown service).  

I'm sorry if this comes off as a rant, but I see how much work goes into this project and it kills me that it can only easily be used as a super-featured online sandbox

**Comment by yann300** [comment]
Thanks for the feedback, we'll address this soon.
Also we have another repository ( https://github.com/ethereum/remix-desktop/releases/tag/v1.0.8 ) which host the desktop version.
Unfortunately it doesn't come with metamask (yet..) but you can use a mobile wallet (also mobile metamask).
The desktop version can also be used for working with local files.

**Comment by acvrp-lab** [comment]
so far i've tried 1)nx serve, 2)v0.10.7 release zip
both not working if i turn off net from the begining
2) need to retrieve compiler list from net
1) compiler list shows one 'lastest local version 0.7.4', but cant compile

main reason i want an offline version, is that sometimes it takes long time to download compiler (sometimes it fails). does browers prohibit keeping a local cache of the compiler between sessions?

**Comment by Aniket-Engg** [comment]
Instructions are updated. So closing this.
